examples/requirements.txt

```diff
@@ -0,0 +1,50 @@
+amqp==1.4.9
```

Comment:
@jacquerie please specify only top level requirements and use `>=X.Y,X.Y+1` if possible.